client/components/mobile-phone-change/mobile-phone-change.js

```diff
@@ -119,9 +119,9 @@ class MobilePhoneChange extends React.Component {
           '"response_code" !== "AUTH_TOKEN_VALIDATION_SUCCESSFUL"',
         );
       } else {
-        const {phone_number, is_active} = response.data;
+        const {phone_number} = response.data;
         this.setState({phone_number});
-        verifyMobileNumber(!is_active);
+        verifyMobileNumber(settings.mobile_phone_verification);
```

Comment:
`settings.mobile_phone_verification` is checked elsewhere, just do `verifyMobileNumber(true)`, and add a preceding comment like "Changing the phone number always requires verification".